llvm/lib/Target/RISCV/RISCVTargetTransformInfo.cpp

```diff
@@ -1179,6 +1179,79 @@ RISCVTTIImpl::getIntrinsicInstrCost(const IntrinsicCostAttributes &ICA,
     return getCmpSelInstrCost(Instruction::Select, ICA.getReturnType(),
                               ICA.getArgTypes()[0], CmpInst::BAD_ICMP_PREDICATE,
                               CostKind);
+  case Intrinsic::vp_reduce_add:
```

Comment:
Please see CachingVPExpander::expandPredicationInReduction as an example of how to perform these mappings more cleanly.

Reply:
Updated, thanks!

Reply:
Please use:

```
Intrinsic::ID RedID = *VPI.getFunctionalIntrinsicID();
unsigned Opc = getArithmeticReductionInstruction(RedID);
OR
getMinMaxReductionIntrinsicOp(RedID);
```

Reply:
Fixed, thanks!!